microprofile/microprofile-jwt-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/jwt/v11/MPConfigLocationJwt11Test.java

```diff
@@ -0,0 +1,30 @@
+package org.wildfly.swarm.ts.microprofile.jwt.v11;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.ClassLoaderAsset;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class MPConfigLocationJwt11Test extends AbstractMPConfigJwt11Test {
+
+    @Deployment
+    public static Archive<?> deployment() {
+        String publicKey = "loadedPublicKey";
+        String mpConfigProperties = "mp.jwt.verify.publickey.location=" + publicKey + "\n" +
+                "mp.jwt.verify.issuer=" + AbstractMPConfigJwt11Test.ISSUER;
```

Comment:
This should be a file in `src/test/resources` and a `ClassLoaderAsset` instead of a `String`/`StringAsset`.